tests/selenium_ex_tests/test_selenium_ex1.py

```diff
@@ -0,0 +1,7 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
```

Comment:
In python, we need to add 2 new lines after the imports and the rest of the code.
Also, for future references, we need to add another 2 new lines between methods/functions.